### PR TITLE
Add new banner

### PR DESCRIPTION
**Pull request by albertcui**

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -150,7 +150,12 @@ const App = (props) => {
         <Helmet
           defaultTitle={strings.title_default}
           titleTemplate={strings.title_template}
-        />
+        >
```
Comment (howardchung) [inline src/components/App/App.jsx:153]: isn't this the top?

**Comment by howardchung** [comment]
Should also probably update https://github.com/odota/web/blob/master/.env

Netlify preview should also let you see if it worked: https://deploy-preview-2592--opendota-staging.netlify.app/

File: src/components/App/App.jsx (hunk 1 above)
Comment (albertcui) [reply src/components/App/App.jsx:153]: Yes, we have to add a script to the header and to where we want it to render.

**Comment by albertcui** [comment]
> Should also probably update https://github.com/odota/web/blob/master/.env
> 
> Netlify preview should also let you see if it worked: https://deploy-preview-2592--opendota-staging.netlify.app/

We need to get verified before anything actually will work, so we basically need to get this deployed and wait a few days...

**Comment by howardchung** [comment]
OK, do you remember how to deploy?

**Pull request merged**